### DISCUSSION TITLE
How do i use Image Input with gpt-4o model ?

### DISCUSSION THREAD

**Discussion by khanakia** [janhq/jan#3077] [Get Help] [answered]
I am trying to upload the JPEG image but got the below erorr

But openai supports Image Input https://help.openai.com/en/articles/8400551-image-inputs-for-chatgpt-faq

<img width="1472" alt="Screenshot 2024-06-21 at 8 39 59 AM" src="https://github.com/janhq/jan/assets/2174170/ffc36373-0d08-4d6c-8c59-7aea857f04e8">

**Comment by kalle07** [comment]
model
OpenAI GPT 4 with Vision (Preview)

but you need api-key

**Reply by khanakia** [reply]
Same error and also GPT 4 Vision is depreciated

**Reply by kalle07** [reply]
you press these Paper clip (attchement symbol) and choose an image?

so these mode has the capability but this means not it work ...

for me only a local model works 
koboldcpp (you know?)

**Reply by khanakia** [reply]
Check this image is disabled 
![Screenshot 2024-06-22 at 5 41 04 PM](https://github.com/janhq/jan/assets/2174170/eb6794c7-0461-45e8-95cf-c90920253e79)

**Reply by kalle07** [reply]
for me at least its open
but i have no apikey to check ;)
![grafik](https://github.com/janhq/jan/assets/118767589/04cf77e0-e3fa-494f-b190-82fb7d7df1f6)



local models dosnt work for me...

only kobold works localy for me

**Comment by ux-han** [comment, marked as answer]
Currently it's a bug & will be solved here: https://github.com/janhq/jan/issues/3520
